service-onboarding-build-pack/Jenkinsfile

```diff
@@ -643,3 +641,18 @@ def loadConfigModule(buildModuleUrl){
 		echo "finished loading env module"
 	}
 }
+
+/*
+* Load environment variables from build module
+*/
+def loadSCMModule(buildModuleUrl){
+	dir('config-loader') {
+		checkout([$class: 'GitSCM', branches: [
+			[name: '*/master']
+		], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [
+			[credentialsId: repo_credential_id, url: buildModuleUrl]
+		]])
+
+		scmModule = load "scm-module.groovy"
```

Comment:
We need to initialize the module with config data?

Reply:
Yes.